docs/FAQ.md

```diff
@@ -11,33 +11,39 @@ See the [rotation distance document](Rotation_Distance.md).
 
 ## Where's my serial port?
 
-The general way to find a USB serial port is to run `ls
-/dev/serial/by-id/*` from an ssh terminal on the host machine. It will
-likely produce output similar to the following:
+The general way to find a USB serial port is to run
+``` shell
+ls /dev/serial/by-id/*
 ```
+from an ssh terminal on the host machine. It will
+likely produce output similar to the following:
+``` shell
 /dev/serial/by-id/usb-1a86_USB2.0-Serial-if00-port0
 ```
 
-The name found in the above command is stable and it is possible to
-use it in the config file and while flashing the micro-controller
-code. For example, a flash command might look similar to:
-```
+The name found in the above command is stable and it is possible to use
+it in the config file and while flashing the micro-controller code.
+For example, a flash command might look similar to:
+``` shell
 sudo service klipper stop
 make flash FLASH_DEVICE=/dev/serial/by-id/usb-1a86_USB2.0-Serial-if00-port0
 sudo service klipper start
 ```
 and the updated config might look like:
-```
+``` yaml
```

Comment:
It seems odd to mark "ini" file content as "yaml".

Reply:
fixed - sry I guessed from the vscode code highlight which one it could be.